test/api-test/api/v2/search/jobcodes/get.js

```diff
@@ -2,6 +2,8 @@
 
 pm.test("Check job code (occupation) search", function () {
   let responseData = pm.response.json();
-
-  pm.expect(responseData).to.have.deep.equal(expectedData);
-});
\ No newline at end of file
+  // console.log(expectedData);
+  // console.log(responseData);
+  // pm.expect(responseData).to.have.deep.equal(expectedData);
```

Comment:
The theory was that these assertions were failing because of case sensitive search results for the metadata types (job codes, and all 7 values implemented as "keyword" (keyword, category, alignment, etc.)

To confirm, @shnooga and @psanchez-wgu, this test change means we are accepting a reduction in capabilities and test coverage for OSMT. We have been testing the actual contents of the search results, for result size, and also sort order. This change now only covers the size of the results, and the sort order is not covered.

If we are OK with this change, then we can remove the commented code in these Postmaster test files. If we are holding out for case insensitive sorting, then we have a little more work to do. I tagged the OsmtQueryHelper class with a corresponding remark, but look at the existing code for where we structured both the ES index (the storage) for case insensitivity, and also the queries themselves.